packages/apps/reputation-oracle/server/src/modules/user/user.service.ts

```diff
@@ -189,33 +151,31 @@ export class UserService {
     data.address = data.address.toLowerCase();
```

Comment:
Would be better to not change the input, but have a separate variable and use it:
```
const addressToRegister = data.address.toLowerCase();
```